README.md

```diff
@@ -1,13 +1,13 @@
 # Project Auth
 
-Replace this readme with your own information about your project.
+In this project we have build an API with authentication to implement a registration flow, and a frontend with a forms to register, sign in, and once you're logged in you can view the secret message.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Tech
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Express, React, React Router, Redux, MogoDB/Mongoose, Styled Components, Node.
 
 ## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Frontend:
+https://practical-wozniak-d60eff.netlify.app/Login 
```

Comment:
A tip! Remove "login" from the URL, otherwise you'll get an error (404) when opening the link